messaging/README.md

```diff
@@ -1,22 +1,38 @@
-Firebase Cloud Messaging Quickstart
-===================================
+# Firebase Cloud Messaging Quickstart
 
 The Firebase Cloud Messaging quickstart demonstrates how to:
+
 - Request permission to send app notifications to the user.
 - Receive FCM messages using the Firebase Cloud Messaging JavaScript SDK.
 
-Introduction
-------------
+## Introduction
+
+### Sending
```

Comment:
Is there a need to know this before running the quickstart?  Does the quickstart send only a notification, or does it demonstrate data messages as well?

I can see providing this information in the References section below, going specifically to "Message types" https://firebase.google.com/docs/cloud-messaging/concept-options#notifications_and_data_messages.